platform/ui/src/components/Label/Label.jsx

```diff
@@ -1,32 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 
-const Label = ({ children, spacing, className, text, ...rest }) => {
-  const baseClasses = 'flex flex-1 flex-col text-gray-700 text-sm font-bold';
-
-  const getClasses = () => {
-    const classes = [];
-    classes.push(baseClasses);
-    classes.push(className);
-    return classes.join(' ');
-  };
+const Label = ({ children, className, text, ...rest }) => {
```

Comment:
No rest spread. Should probably add an eslint rule. @rodrigolabs, is there one floating around in the react eslint rules we can steal?

Reply:
#1509